src/app/pages/join-session/join-session.component.ts

```diff
@@ -14,7 +14,9 @@ export class JoinSessionComponent implements OnInit {
 
   sessionFormGroup = new FormGroup({
     nameControl: new FormControl('', Validators.required),
-    sessionControl: new FormControl('', Validators.required)
+    sessionControl: new FormControl('', Validators.required),
+    nestControl: new FormControl('', [Validators.required, Validators.min(18), Validators.max(127),
```

Comment:
This should still be 20-120, it seems nice to have a max of exactly 100 nests/students.
```suggestion
    nestControl: new FormControl('', [Validators.required, Validators.min(20), Validators.max(120),
```